docs/source/tutorials/filtsmooth/linear_bayesian_filtering_smoothing.ipynb

```diff
@@ -0,0 +1,4552 @@
+{
```

Comment:
<p>linear, time-invariant</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>For the sake of consistency, I would prefer that to stay capitalized, if that's ok with you.</p>

Reply:
<p>I did not mean capitalisation, but the comma and the dash: Linear, Time-Invariant</p>

Reply:
💡 Alright :)